tachyon/c/zk/plonk/halo2/test/bn254_halo2_params_data.h

```diff
@@ -0,0 +1,304 @@
+#ifndef TACHYON_C_ZK_PLONK_HALO2_TEST_BN254_HALO2_PARAMS_DATA_H_
+#define TACHYON_C_ZK_PLONK_HALO2_TEST_BN254_HALO2_PARAMS_DATA_H_
+
+#include <stddef.h>
```

Comment:
I'm not sure if this header is being used.

Reply:
The header is for `size_t`.

Reply:
I've confirmed that `size_t` is not used in this file!